tests/dummy/app/locales/ru/translations.js

```diff
@@ -296,7 +296,8 @@ Ember.$.extend(true, translations, {
       'name-caption': 'Наименование',
       'moderated-caption': 'Одобрено',
       'parent-caption': 'Иерархия',
-      'localizedTypes-caption': 'Локализация типа'
+      'localizedTypes-caption': 'Локализация типа',
```

Comment:
I suppose we should use single keys style for all keys in i18n translations files: kebab case instead of camel case (localized-types-caption instead of localizedTypes-caption).